src/http/lib/WebServer/WebServerSingleton.ts

```diff
@@ -14,21 +14,21 @@ export default class WebServerSingleton {
    * This implementation let you subclass the Singleton class while keeping
    * just one instance of each subclass around.
    */
-  public static async getInstance(): Promise<WebServer> {
+  public static async getInstance(withoutInit:boolean=false): Promise<WebServer> {
```

Comment:
I wouldn't introduce a new parameter. Instead I would rather create an init method that has to be called to define that the Dapr Client was started. Before that we would then be able to register routes.

E.g.

```javascript
const client = new Dapr();
await client.pubsub.listen();
await client.start();
```